applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -194,7 +194,7 @@ static void hidc_ready(struct bt_gatt_hids_c *hids_c)
 
 	while (NULL != (rep = bt_gatt_hids_c_rep_next(hids_c, rep))) {
 		if (bt_gatt_hids_c_rep_type(rep) ==
-		    BT_GATT_HIDS_C_REPORT_TYPE_INPUT) {
+		    BT_GATT_HIDS_COMMON_REPORT_TYPE_INPUT) {
```

Comment:
Use rather BT_GATT_HIDS_REPORT_TYPE_INPUT. Remove the COMMON word - it provides no information only making it harder to understand.